src/n2o_proto.erl

```diff
@@ -45,6 +45,7 @@ init(_Transport, Req, _Opts, _) ->
     put(context,Zero),
     Ctx  = fold(init,Zero#cx.handlers,Zero),
     put(context,Ctx),
+    save_context(Ctx),
```

Comment:
do we need here put and save_context both ?

Reply:
Yes.